src/main/java/complete_task/completeTaskOutputBoundary.java

```diff
@@ -9,5 +10,7 @@
 public interface completeTaskOutputBoundary {
 
     int taskId = completeTaskController.taskId;
-    boolean taskCompleted(int taskId, Curriculum taskCurriculum);
+    int taskCurriculumId = completeTaskController.taskCurriculumId;
```

Comment:
same notes about these attributes

Reply:
I hadn't caught this! All the attributes should be in the implementation; only method signatures should exist in the interface level. If you look at my code for the implemented Task classes, for example, you'll see that the CommonTask class is the only thing with attributes like name or completion; the interface is meant only to describe the behavior to those who depend on it.